playbooks/roles/caasp4/tasks/_assert_packages.yml

```diff
@@ -0,0 +1,73 @@
+---
+#
+# (c) Copyright 2019 SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+- name: Identify localhost SUSE OS distribution and version
+  set_fact:
+    suse_repo_version: "CAASP40-{{ansible_distribution}}-{{ansible_distribution_version | replace('.', '') }}"
+  when:
+    - ansible_distribution | lower == "sles"
+
+- name: Configure repositories for skuba and terraforms
+  become: yes
+  zypper_repository:
+    name: "{{ suse_repo_version }}"
+    repo: "{{ socok8s_repos_to_configure[suse_repo_version] }}"
+    autorefresh: True
+    auto_import_keys: yes
+    state: present
+  when:
+    - suse_repo_version is defined
+    - suse_repo_version in socok8s_repos_to_configure
+
+- name: Install required skuba, terraform packages for OS versions
+  become: yes
+  package:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - skuba
+    - terraform
+  when:
+    - suse_repo_version is defined
+    - suse_repo_version in socok8s_repos_to_configure
+
+- name: Check skuba binary is available
```

Comment:
we need to check that the *package* is installed, because that is what installs the proper templates to the proper place. you can probably use `package_facts` here

Reply:
Yep, good idea as I did not knew about it earlier. Looks like it cannot be used.

Unfortunately it does not work for SLES as I am seeing following error 
`"msg": "Could not detect a supported package manager from the following list: ['pkg', 'apt', 'portage', 'rpm']"
`

When specified 'rpm' as manager for package_facts module, it gave following error.
`"msg": "Could not detect a supported package manager from the following list: ['rpm']"
`